drivers/SmartThings/harman-luxury/profiles/maX10.yaml

```diff
@@ -0,0 +1,29 @@
+name: maX10
+components:
+- id: main
+  capabilities:
+  - id: switch
```

Comment:
Is there somewhere I can base the order upon or any requirements on that matter?
I personally didn't notice anything weird inthe app

Reply:
Unfortunately there isn't anything I know of where its defined, but since you say everything is showing up properly in the app, this is probably just something of the past.